[2022-10-09 02:33:41,258] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: recruitment-airflow.KafkaConnectTest scheduled__2022-10-08T00:00:00+00:00 [queued]>
[2022-10-09 02:33:41,288] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: recruitment-airflow.KafkaConnectTest scheduled__2022-10-08T00:00:00+00:00 [queued]>
[2022-10-09 02:33:41,289] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-10-09 02:33:41,290] {taskinstance.py:1242} INFO - Starting attempt 1 of 1
[2022-10-09 02:33:41,291] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-10-09 02:33:41,365] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): KafkaConnectTest> on 2022-10-08 00:00:00+00:00
[2022-10-09 02:33:41,368] {standard_task_runner.py:52} INFO - Started process 31 to run task
[2022-10-09 02:33:41,371] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'recruitment-airflow', 'KafkaConnectTest', 'scheduled__2022-10-08T00:00:00+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/recruitment_pipeline.py', '--cfg-path', '/tmp/tmpew1qohtl', '--error-file', '/tmp/tmpg_bpulqk']
[2022-10-09 02:33:41,373] {standard_task_runner.py:77} INFO - Job 86: Subtask KafkaConnectTest
[2022-10-09 02:33:41,463] {logging_mixin.py:109} INFO - Running <TaskInstance: recruitment-airflow.KafkaConnectTest scheduled__2022-10-08T00:00:00+00:00 [running]> on host pipeline
[2022-10-09 02:33:41,512] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/xcom.py:323 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-09 02:33:41,534] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/renderedtifields.py:126 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-09 02:33:41,579] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/renderedtifields.py:162 SAWarning: Coercing Subquery object into a select() for use in IN(); please pass a select() construct explicitly
[2022-10-09 02:33:41,583] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/renderedtifields.py:159 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-09 02:33:41,605] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=recruitment-airflow
AIRFLOW_CTX_TASK_ID=KafkaConnectTest
AIRFLOW_CTX_EXECUTION_DATE=2022-10-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-08T00:00:00+00:00
[2022-10-09 02:33:41,606] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-10-09 02:33:41,607] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'python /opt/pipeline/script/KafkaConnectTest.py']
[2022-10-09 02:33:41,612] {subprocess.py:85} INFO - Output:
[2022-10-09 02:33:47,242] {subprocess.py:89} INFO - 1번 째 연결 실패
[2022-10-09 02:33:47,243] {subprocess.py:89} INFO - 2번 째 연결 성공
[2022-10-09 02:33:47,253] {subprocess.py:93} INFO - Command exited with return code 0
[2022-10-09 02:33:47,260] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/xcom.py:103 SAWarning: TypeDecorator UtcDateTime(timezone=True) will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object's state is safe to use in a cache key, or False to disable this warning. (Background on this error at: https://sqlalche.me/e/14/cprf)
[2022-10-09 02:33:47,324] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=recruitment-airflow, task_id=KafkaConnectTest, execution_date=20221008T000000, start_date=20221009T023341, end_date=20221009T023347
[2022-10-09 02:33:47,422] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-09 02:33:47,530] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-10-09 02:33:47,533] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:1069 SADeprecationWarning: Query.value() is deprecated and will be removed in a future release.  Please use Query.with_entities() in combination with Query.scalar() (deprecated since: 1.4)
[2022-10-09 02:33:47,535] {logging_mixin.py:109} WARNING - /usr/local/lib/python3.8/site-packages/airflow/models/dag.py:1069 SADeprecationWarning: Query.values() is deprecated and will be removed in a future release.  Please use Query.with_entities() (deprecated since: 1.4)
